ipfx/dataset/ephys_data_interface.py

```diff
@@ -189,15 +162,13 @@ def get_stimulus_unit(self, sweep_number: int) -> str:
         raise NotImplementedError
 
     @abc.abstractmethod
-    def get_real_sweep_number(self, sweep_name:str, assumed_sweep_number: Optional[int]=None) -> int:
+    def get_sweep_number(self, sweep_name:str) -> int:
```

Comment:
can this be deleted?

Reply:
I believe so. The logic was to try getting the "real" (explicitly assigned) sweep number. But for old nwb1 files it was missing, in which case resort to inferring sweep number from the dataset name. My understanding that for nwb2 "real" sweep number is always exists.

Reply:
ok, would you mind deleting then?

Reply:
I thought we were talking about `get_real_sweep_number`, but you want me to delete `get_sweep_number`. got it.